tachyon/crypto/commitments/kzg/kzg_unittest.cc

```diff
@@ -0,0 +1,48 @@
+#include "tachyon/crypto/commitments/kzg/kzg.h"
+
+#include "gtest/gtest.h"
+
+#include "tachyon/math/elliptic_curves/bn/bn254/g1.h"
+#include "tachyon/math/elliptic_curves/bn/bn254/g2.h"
+#include "tachyon/math/polynomials/univariate/radix2_evaluation_domain.h"
+
+namespace tachyon::crypto {
+
+namespace {
+
+class KZGTest : public testing::Test {
+ public:
+  using UnivariateEvaluationDomainTy =
```

Comment:
Please create the evaluation domain through a factory. See, univariate_evaluation_domain_factory.h.